libr/core/panels.c

```diff
@@ -286,7 +286,7 @@ static void set_root_state(RCore *core, RPanelsRootState state);
 static void resetScrollPos(RPanel *p);
 
 /* update */
-static void updateDisassemblyAddr (RCore *core);
+static void update_disassembly_or_open (RCore *core);
```

Comment:
according to the devrules, static(private) functions should start with double lower dash

Reply:
you should've told me this at day one but ok. and also, wanna delete space after the name. hang on

Reply:
way too many functions ignoring this rule in the whole code base. do you unify them into this rule after all?